src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,36 @@
+import errorIcon from "@assets/images/error.svg";
+import warningIcon from "@assets/images/warning.svg";
+import { Tooltip } from "@saleor/macaw-ui";
+import React from "react";
+import SVG from "react-inlinesvg";
+
+export type BadgeVariant = "warning" | "error";
+
+export interface StatusBadgeProps {
+  variant: BadgeVariant;
+  message: string;
```

Comment:
Would rather call it description, looking back at our component that after some time gain e.g. title, then it'd remain consistent if we add another types of texts

Reply:
Will do